code/src/index.css

```diff
@@ -1,13 +1,280 @@
+*, *::before, *::after { box-sizing: border-box; }
+
 body {
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
+  font-family: 'Josefin Slab', serif;
+  background-color: #74583E;
+  color: #FFF7EE;
+}
+
+header {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 300px;
+  padding-top: 300px;
+  background-color: #5E6572;
+  font-family: 'Sacramento', cursive;
+}
+
+.celebrate {
+  align-self: center;
+  padding-top: 40px;
+  padding-bottom: 150px;
+  width: 100px;
+}
+
+.arrow {
+  height: 100px;
+  padding-top: 50px;
+}
+
+.bounce-arrow {
+  align-self: center;
+	animation: bounce 2s;
+	animation-iteration-count: infinite;
+}
+
+@keyframes bounce {
+	0%, 100% 
+  { transform: translateY(0); }
+  50% 
+  { transform: translateY(-20px); }
+}
+  
+.ballons {
+  width: 180px;
+}
+
+h1 {
+  margin: 0;
+  font-size: 47px;
+}
+
+h2 {
+  font-size: 24px;
+  margin: 0;
+}
+
+h4 {
+  font-size: 24px;
+}
+
+p {
+  font-size: 18px;
+}
+
+.invitation {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
+  height: 700px;
+  padding-top: 80px;
+  padding-left: 20px;
+  padding-right: 20px;
+  line-height: 1.5;
+  background-color: #5E6572;
+}
```

Comment:
Consistent structure on where you have what for each class and html element, makes it so easy for me to follow the css code - that you start with flexbox, then go to height/width and font for example. GREAT!